README.md

```diff
@@ -1 +1,59 @@
-# angular-project-setup-lab
+# Project Setup
+
+## Objectives
+
+- Download the latest version of Angular 1.x
+- Setup the directory structure for your Angular application
+ 
+## Instructions
+
+Go to the [Angular website](https://angularjs.org/) and download the latest version.
+
+Setup your directory structure. From the root directory, the structure should be as follows (you can find the index.html in this repo) -
+
+- js/
+  - app/
+    - directives/
+    - controllers/
+    - services/
+    - templates/
+      - partials/
+      - views/
+  - angular.js
+- img/
+- css/
+- index.html
+
+Make sure you put the Angular code you downloaded into the `js/` folder!
+
+Let's create some files so we can have something pretty displayed in browser. No need to worry with what this code does yet, we will be covering it all very shortly!
+
+### app.js
+
+Inside the `js/app/` folder, create a new file called `app.js`. This file is very simple, and again - no need to worry about what it does just yet!
```

Comment:
Here, I think it's okay to give a one-line explanation of what this file is doing. When learning Sinatra, we tell students that the `config.ru` file is the one that kicks off the application. At this point, they can start to get an idea of the flow of the app.